source/dub/dub.d

```diff
@@ -167,6 +167,47 @@ class Dub {
 
 		init();
 
+		if (skip_registry == SkipPackageSuppliers.none)
+			m_packageSuppliers = getPackageSuppliers(additional_package_suppliers);
+		else
+			m_packageSuppliers = getPackageSuppliers(additional_package_suppliers, skip_registry);
+
+		m_packageManager = new PackageManager(m_dirs.localRepository, m_dirs.systemSettings);
+
+		auto ccps = m_config.customCachePaths;
+		if (ccps.length)
+			m_packageManager.customCachePaths = ccps;
+
+		updatePackageSearchPath();
+	}
+
+	unittest
+	{
+		scope (exit) environment.remove("DUB_REGISTRY");
+		auto dub = new Dub(".", null, SkipPackageSuppliers.configured);
+		assert(dub.m_packageSuppliers.length == 0);
+		environment["DUB_REGISTRY"] = "http://example.com/";
+		dub = new Dub(".", null, SkipPackageSuppliers.configured);
+		logInfo("%s", dub.m_packageSuppliers);
```

Comment:
I would probably remove this to avoid polluting the output when `dub test` is run.

Reply:
fixed :)